stable_baselines/acer/acer_simple.py

```diff
@@ -434,11 +435,19 @@ def _train_step(self, obs, actions, rewards, dones, mus, states, masks, steps, w
         td_map = {self.train_model.obs_ph: obs, self.polyak_model.obs_ph: obs, self.action_ph: actions,
                   self.reward_ph: rewards, self.done_ph: dones, self.mu_ph: mus, self.learning_rate_ph: cur_lr}
 
+        num_steps = self.train_model.action_mask_ph.shape[0]
+        if len(action_masks) == 0:
+            action_masks = create_dummy_action_mask(self.train_model.ac_space, num_steps)
+        else:
+            action_masks = reshape_action_mask(action_masks, self.train_model.ac_space, num_steps)
+
         if states is not None:
             td_map[self.train_model.states_ph] = states
             td_map[self.train_model.dones_ph] = masks
             td_map[self.polyak_model.states_ph] = states
             td_map[self.polyak_model.dones_ph] = masks
+        td_map[self.train_model.action_mask_ph] = action_masks
+        td_map[self.polyak_model.action_mask_ph] = action_masks
```

Comment:
Not a fan of accessing the tensor like this, but I didn't see another way to do it cleanly besides make a member variable for it, but that duplicates data